ui-backend/intrigue-ui-app/src/main/resources/features.xml

```diff
@@ -85,7 +86,7 @@
 
     <feature name="javalin" version="${project.version}" description="Javalin Web Framework">
         <feature prerequisite="true">wrap</feature>
-        <bundle>wrap:mvn:org.checkerframework/checker-qual/2.0.0$Bundle-Name=checker-qual&amp;Bundle-SymbolicName=checker-qual&amp;Bundle-Version=2.0.0&amp;Export-Package=org.checkerframework.checker.nullness.qual;version="2.0.0"</bundle>
+<!--        <bundle>wrap:mvn:org.checkerframework/checker-qual/2.0.0$Bundle-Name=checker-qual&amp;Bundle-SymbolicName=checker-qual&amp;Bundle-Version=2.0.0&amp;Export-Package=org.checkerframework.checker.nullness.qual;version="2.0.0"</bundle>-->
```

Comment:
This was giving me trouble on install. Will need to revisiti

Reply:
Looks to be working - will uncomment for the release
<img width="1672" alt="image" src="https://github.com/user-attachments/assets/ea703ea4-1600-4482-ab65-b414b5a466bb" />